packages/headless/src/controllers/product-listing/sort/headless-product-listing-sort.ts

```diff
@@ -41,23 +41,17 @@ export {
   buildFieldsSortCriterion,
 };
 
-export interface ProductListingSort extends Controller {
-  sortBy(criterion: SortCriterion): void;
-
-  isSortedBy(criterion: SortCriterion): boolean;
-
-  state: ProductListingSortState;
-}
-
-export interface ProductListingSortState {
-  sort: SortCriterion;
-}
-
 export interface ProductListingSortProps {
+  /**
+   * The initial state that should be applied to this `Sort` controller.
```

Comment:
Smaller detail, but with `Sort` I believe there is only a single state property, even if multiple controllers are instantiated?
If so, I think `this Sort controller` could be misleading. Would the following be more accurate?

```
The initial sort state to apply.
```

Reply:
I agree, but it's the exact same line as in `headless-sort`:

> The initial state that should be applied to this `Sort` controller.

:grimacing: